common/scripted_triggers/iisc_triggers.txt

```diff
@@ -67,22 +67,33 @@ iisc_should_build_science_ship = {
 		which = iisc_budget_science_ship
 		value >= @iisc_science_ship_cost
 	}
-	count_owned_ship = {
-		limit = {
-			is_ship_size = science
+	iisc_has_not_enough_science_ships = yes
+}
+
+# this = corp
+iisc_has_not_enough_science_ships = {
+	OR = {
+		count_owned_ship = {
+			limit = {
+				is_ship_size = science
+			}
+			count < 10
+		}
+		AND = {
+			has_civic = iisc_civic_exploration_initiative
+			count_owned_ship = {
+				limit = {
+					is_ship_size = science
+				}
+				count < 20
+			}
 		}
-		count < 5		# TODO change
 	}
 }
 
 # this = corp
 iisc_should_destroy_science_ship = {
-	count_owned_ship = {
-		limit = {
-			is_ship_size = science
-		}
-		count > 5		# TODO change
-	}
+	always = no
```

Comment:
Perhaps TODO, but once the corp is running out of things to survey we may want to reduce the number of science ships.

Reply:
Ideally the science ships will have something other to do 🤔